ksp_plugin/part.cpp

```diff
@@ -89,6 +91,14 @@ Mass const& Part::mass() const {
   return mass_;
 }
 
+void Part::set_centre_of_mass(Position<EccentricPart> const& centre_of_mass) {
+  centre_of_mass_ = centre_of_mass;
+}
+
+RigidMotion<RigidPart, EccentricPart> Part::ToEccentric() const {
```

Comment:
I don't find the name very clear.  Maybe `rigid_to_eccentric_motion` would read better at the call site?

Also, I would invest in a member variable (and serialize it).

Reply:
As discussed, `MakeRigidToEccentricMotion`, and no.